src/app/dangermessage/dangermessage.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+class danger{
+  danger:string;
+  
+  constructor(danger:string){
+  
+    this.danger = danger;
+  }
+  
+  }
+@Component({
+  selector: 'dangermessage',
+  templateUrl: './dangermessage.component.html',
+  template:`
```

Comment:
you cannot have both templateUrl and template defined for the meta-data here. you need to mention only one